app/subscriber/src/features/login/BrowserLogin.tsx

```diff
@@ -75,12 +88,7 @@ export const BrowserLogin: React.FC<IBrowserLoginProps> = ({ login }) => {
             <div className="alert-containing-box">
               <b>System Notices</b>
               <br />
-              <p>
-                Service interruption Dec 29 - Jan 3. Stories will not be updated over the holidays.{' '}
-                <br />
-                <br />
-                Jan 4th - Updates resume, business as usual.
-              </p>
+              <p>{!!alert?.message ? alert.message : 'No alert messages at this time.'}</p>
```

Comment:
I doubt they want a default message.  Just leave it blank if there is no message.